examples/textual_inversion/textual_inversion.py

```diff
@@ -679,13 +678,11 @@ def main():
             save_full_model = not args.only_save_embeds
         if save_full_model:
             pipeline = StableDiffusionPipeline(
+                args.pretrained_model_name_or_path,
                 text_encoder=accelerator.unwrap_model(text_encoder),
                 vae=vae,
                 unet=unet,
                 tokenizer=tokenizer,
-                scheduler=PNDMScheduler.from_pretrained(args.pretrained_model_name_or_path, subfolder="scheduler"),
-                safety_checker=StableDiffusionSafetyChecker.from_pretrained("CompVis/stable-diffusion-safety-checker"),
-                feature_extractor=CLIPFeatureExtractor.from_pretrained("openai/clip-vit-base-patch32"),
             )
```

Comment:
Hmm, I think as is it might throw an error.

I think this is what we're going for?
```
            pipeline = StableDiffusionPipeline(
                text_encoder=accelerator.unwrap_model(text_encoder),
                vae=vae,
                unet=unet,
                tokenizer=tokenizer,
                scheduler=noise_scheduler,
                safety_checker=None,
                feature_extractor=None,
                requires_safety_checker=False,
            )
            pipeline.save_pretrained(args.output_dir)
```

Reply:
Ah yes, I forgot the `from_pretrained(...)`